### PR TITLE
html: Add tests to verify BigInt structured clone integration

### PR DESCRIPTION
**Pull request by littledan**
These tests are against a PR which has not yet landed: https://github.com/whatwg/html/pull/3480

### CODE REVIEW

**Comment by domenic** [comment]
It might be good to put this in a separate file, if it's not too much trouble, so that browsers which don't implement the new syntax don't start suddenly failing all existing tests due to a SyntaxError in the same file.

**Comment by w3c-bots** [comment]
# Build PENDING

Started: None
Finished: None





View more information about this build on:

- [WPT PR Status](http://pulls.web-platform-tests.org/build/24497)
- [TravisCI](https://travis-ci.org/w3c/web-platform-tests/builds/342862398)

**Comment by littledan** [comment]
@domenic Good idea, split out into a separate file. Unfortunately, this is untested copypasta at this point, to be clear.

**Comment by littledan** [comment]
These tests are missing verification that BigInt and BigInt wrappers can be used as IndexedDB values (but not (yet) keys), although the PR in https://github.com/whatwg/html/pull/3480 permits them to be used as such. The testing will come in a separate PR (once I figure out any other changes for BigInt that IndexedDB might need).

File: html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html
Comment (domenic) [inline html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html]: Fix this to be https://html.spec.whatwg.org/multipage/#safe-passing-of-structured-data

File: html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html
Comment (domenic) [inline html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html]: Maybe add a note that this is supposed to be similar to its sibling, but is separate to avoid JS syntax errors? That'll help people who wonder at the strange style of the test.

**Review by domenic — APPROVED** [review]
This could definitely be simplified and modernized but I think as long as we add a comment explaining that it's just aping its sibling this'll do fine.

File: html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html
Comment (littledan) [reply html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html]: Done

File: html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html
Comment (littledan) [reply html/infrastructure/safe-passing-of-structured-data/structured_clone_bigint.html]: Done

**Comment by littledan** [comment]
What's the convention here--should I land this myself now that it's been approved (seems like I have permissions), or wait for a maintainer to land it?

**Comment by annevk** [comment]
You should feel free to land it yourself in the future. Please do include links to the standard change PR in the commit message as I did when landing. Makes it easier to understand things in the future.

**Pull request merged**
